src/qililab/result/database/database_autocal.py

```diff
@@ -0,0 +1,179 @@
+                return persistent_instance
+            except Exception as e:
+                session.rollback()
+                raise e
+
+    def __repr__(self):
+        return f"calibration_run: {self.calibration_id} {self.date}"
+
+
+class Autocal_Measurement(base):  # type: ignore
+    """Creates and manipulates Measurement metadata database"""
+
+    __tablename__ = "measurements"
+
+    measurement_id: Column = Column("measurement_id", Integer, primary_key=True)
+    experiment_name: Column = Column("experiment_name", String, nullable=False)
+    start_time: Column = Column("start_time", DateTime, nullable=False)
+    end_time: Column = Column("end_time", DateTime)
+    run_length: Column = Column("run_length", Interval)
+    experiment_completed: Column = Column("experiment_completed", Boolean, nullable=False)
+    cooldown: Column = Column("cooldown", String, index=True)
+    sample_name: Column = Column("sample_name", String, nullable=False)
+    calibration_id: Column = Column("calibration_id", ForeignKey(Calibration_run.calibration_id), nullable=False)
+    result_path: Column = Column("result_path", String, unique=True, nullable=False)
+    fitting_path: Column = Column("fitting_path", String, unique=True, nullable=True)
+    qbit_idx: Column = Column("qbit_idx", Integer)
+    platform_after: Column = Column("platform_after", JSONB)
+    platform_before: Column = Column("platform_before", JSONB)
+    qprogram: Column = Column("qprogram", JSONB)
+    calibration: Column = Column("calibration", JSONB)
+    parameters: Column = Column("parameters", JSONB)
+    data_shape: Column = Column("data_shape", ARRAY(Integer))
+
+    def __init__(
+        self,
+        experiment_name,
+        sample_name,
+        calibration_id,
+        result_path,
+        experiment_completed,
+        start_time,
+        qbit_idx,
+        fitting_path=None,
+        cooldown=None,
+        end_time=None,
+        run_length=None,
+        platform_after=None,
+        platform_before=None,
+        qprogram=None,
+        calibration=None,
+        parameters=None,
+        data_shape=None,
+    ):
+        # Required fields
+        self.experiment_name = experiment_name
+        self.sample_name = sample_name
+        self.result_path = result_path
+        self.experiment_completed = experiment_completed
+        self.start_time = start_time
+        self.calibration_id = calibration_id
+        self.qbit_idx = qbit_idx
+
+        # Optional fields
+        self.fitting_path = fitting_path
+        self.cooldown = cooldown
+        self.end_time = end_time
+        self.run_length = run_length
+        self.platform_after = platform_after
+        self.platform_before = platform_before
+        self.qprogram = qprogram
+        self.calibration = calibration
+        self.parameters = parameters
+        self.data_shape = data_shape
+
+        # self.result_array = result_array
+
+    def end_experiment(self, Session, traceback=None):
```

Comment:
```suggestion
    def end_experiment(self, session, traceback=None):
```
lowe case convention for parameters